ext/sqlite3/sqlite3.c

```diff
@@ -160,6 +163,12 @@ PHP_METHOD(sqlite3, open)
 		sqlite3_set_authorizer(db_obj->db, php_sqlite3_authorizer, NULL);
 	}
 
+#if SQLITE_VERSION_NUMBER >= 3026000
+	if (SQLITE3G(dbconfig_defensive)) {
+		sqlite3_db_config(db_obj->db, SQLITE_DBCONFIG_DEFENSIVE, 1, 0);
```

Comment:
Since the last parameter is an `int*` it seems to be better to pass `NULL` instead of `0` for understandability.

Reply:
Sure. Fixed, and squashed the commits.